webextension/background.js

```diff
@@ -9,10 +9,13 @@
 let gCurrentlyPromptingTab;
 
 const Config = (function() {
-  browser.experiments.aboutConfigPrefs.clearPrefsOnUninstall(["enabled"]);
+  browser.experiments.aboutConfigPrefs.clearPrefsOnUninstall(["enabled", "branch"]);
+
+  const UIVariants = ["more-context", "little-context", "no-context"];
 
   class Config {
     constructor() {
+      this._testingMode = true;
```

Comment:
So we're always in testing mode?

Do we even need a special testing mode, if QA can manually set the pref? This part isn't clear to me.

Reply:
Yes, as of this PR we're always in testing mode. I haven't yet figured out yet how to best handle this, but I thought this is fine for the moment.